app/root.tsx

```diff
@@ -67,10 +67,10 @@ export const meta: MetaFunction = ({data}) => {
 export const links: LinksFunction = () => [{rel: 'stylesheet', href: styles}]
 
 export const loader = async ({request}: Parameters<LoaderFunction>[0]) => {
-  const isDomainWithLogo = request.url.match(/ui.stampy.ai/)
-  const isLogoForcedOff = request.url.match(/minLogo/)
-  const isLogoForcedOn = request.url.match(/funLogo/)
-  const minLogo = isDomainWithLogo ? !!isLogoForcedOff : !isLogoForcedOn
+  const isDomainWithFunLogo = request.url.match(/stampy.ai|localhost/) // min logo by default on aisafety.info and 127.0.0.1
+  const isFunLogoForcedOff = request.url.match(/minLogo/)
+  const isFunLogoForcedOn = request.url.match(/funLogo/)
+  const minLogo = isDomainWithFunLogo ? !!isFunLogoForcedOff : !isFunLogoForcedOn
```

Comment:
line 47 will probably also need to be changed :/

Reply:
huh? not sure what you mean, it's currently `${url.origin}/favicon-512.png` so not depending on these conditions...

Reply:
if there is a feature request to make the social preview logo dynamic too, we can do that in future PR, not blocking merge ;)